style.css

```diff
@@ -1,825 +1,810 @@
   }
 
-  .vectormob {
+  .third__img-dsk {
     position: absolute;
-    width: 65px;
-    height: 37px;
-    left: 300px;
-    top: 873px;
+    top: 139px;
+    left: 53px;
+    z-index: 5;
     display: block;
   }
-}
 
-@media (max-width: 565px) {
-  .first-container {
-    padding: 189px 0;
-    background: url(img/pete-bellis-432867-unsplashmob.png) no-repeat;
-    background-size: 565px;
-    background-position: center;
+  .third__block {
+    z-index: 10;
+    width: 475px;
+    height: 351px;
+    top: 255px;
+    margin-right: 183px;
   }
 
-  .header__inner {
-    width: 100%;
-    height: 104px;
-    background: #fff;
-    z-index: 2;
+  .third__container {
+    width: 380px;
+    height: 224px;
   }
 
-  .header__menu {
-    background: #ededed;
-    z-index: 3;
-    height: 104px;
-    width: 104px;
+  .third__title {
+    font-size: 40px;
+    margin-top: 39px;
   }
 
-  .header__menu-item {
-    height: 4px;
-    width: 42.7px;
-    margin-bottom: 7px;
+  .third__subtitle {
+    font-size: 16px;
+    margin-top: 14px;
+    margin-bottom: 33px;
   }
 
-  .header__logo p {
-    margin: 15px 9px;
-    color: black;
-    font-size: 45px;
-    letter-spacing: 4px;
+  .third__text {
+    font-size: 15px;
   }
 
-  .text__container {
-    display: flex;
-    align-items: flex-end;
-    width: 100%;
-    max-width: 750px;
-    height: 677px;
-    letter-spacing: 5px;
+  .third-count {
+    display: inline-block;
+    text-decoration: none;
+    color: #000;
+    font-size: 15px;
+    border: 1.5px solid #000;
+    padding: 12px 32px;
+    margin-left: 47px;
   }
 
-  .text__title {
-    position: inherit;
-    align-items: flex-end;
-    font-size: 73px;
-    line-height: 80px;
-    padding-left: 43px;
-    letter-spacing: 3px;
+  .third__add {
+    border: 1.5px solid #000;
+    padding: 12px 43px;
+    margin-left: 17px;
   }
 
-  .vectormob {
-    position: absolute;
-    width: 55px;
-    height: 30px;
-    left: 45.5%;
-    top: 734px;
-    display: block;
+  .third__next {
+    font-size: 13px;
+    margin-top: 149px;
+    display: inline-block;
+    margin-right: 137px;
+    width: 372px;
+    z-index: 15;
   }
 }
 
-@media (max-width: 490px) {
-  .first-container {
-    padding: 127px 0;
-    background: url(img/pete-bellis-432867-unsplashmob.png) no-repeat;
-    background-size: 490px;
-    background-position: center;
+@media (min-width: 1124px) {
+  .second {
+     grid-template-columns: 601px 397px;
   }
 
-  .header__inner {
-    width: 100%;
-    height: 89px;
-    background: #fff;
-    z-index: 2;
+  .second__bag-dsk {
+     width: 600px;
   }
+}
 
-  .header__menu {
-    background: #ededed;
-    z-index: 3;
-    height: 89px;
-    width: 89px;
-  }
+@media (min-width: 1280px) {
+  .step {
+    position: absolute;
+    width: 100%;
+    display: grid;
+    grid-template-columns: 549px 549px;
+    grid-template-rows: 600px;
+    justify-content: center;
+    top: 168px;
+    z-index: 2;
+    grid-column-gap: 54px;
+}
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/step3/pullRequest?prid=5898150)